src/operator/tensor/elemwise_binary_broadcast_op_basic.cc

```diff
@@ -120,8 +120,13 @@ Example::
    broadcast_mul(x, y) = [[ 0.,  0.,  0.],
                           [ 1.,  1.,  1.]]
 
+Supported sparse operations:
+   broadcast_mul(csr, dense(1D)) = csr
```

Comment:
clarify that this is only supported on CPU

Reply:
Done

Reply:
Done